heft-plugins/heft-sass-plugin/src/SassTypingsGenerator.ts

```diff
@@ -87,6 +92,20 @@ export class SassTypingsGenerator extends StringValuesTypingsGenerator {
       sassConfiguration.exportAsDefault === undefined ? true : sassConfiguration.exportAsDefault;
     const exportAsDefaultInterfaceName: string = 'IExportStyles';
     const fileExtensions: string[] = sassConfiguration.fileExtensions || ['.sass', '.scss', '.css'];
+    const { cssOutputFolder } = sassConfiguration;
+
+    const cssFolder: string | undefined = cssOutputFolder
+      ? path.resolve(buildFolder, cssOutputFolder)
```

Comment:
Should we use `path.join` just in case the output folder has not been generated yet?

Reply:
path.resolve doesn't care whether it exists or not, it just resolves the path to remove relative pathing, ex. `/a/../b` -> `/b`

Reply:
Ah, interesting. I don't know why I thought it actually resolved to a real disk path, but thanks for clarifying and helping me correct a misunderstanding. :)

Reply:
Wait a minute, does this mean that a cssOutputFolder of `'lib'` would not resolve correctly? I feel like the docs here on node.js could be better.
https://nodejs.org/api/path.html#pathresolvepaths

> The given sequence of paths is processed from right to left, with each subsequent path prepended until an absolute path is constructed. For instance, given the sequence of path segments: /foo, /bar, baz, calling path.resolve('/foo', '/bar', 'baz') would return /bar/baz because 'baz' is not an absolute path but '/bar' + '/' + 'baz' is.

Reply:
Yeah, the docs aren't quite clear... In this case, the absolute path would be the `buildFolder` and the `cssOutputFolder` would be appended, since it's not an absolute path.

Reply:
Ah ha ha. I see how I misread the docs and what it actually means now. Thanks. I'll mark this resolved.